### DISCUSSION TITLE
blog/2021-12-05_supermicro_sas2116_flashing/

### DISCUSSION THREAD

**Discussion by giscus** [Sieboldianus/du.nkel.dev#13] [Announcements]
# blog/2021-12-05_supermicro_sas2116_flashing/

A personal code notes blog

https://du.nkel.dev/blog/2021-12-05_supermicro_sas2116_flashing/

**Comment by Sieboldianus** [comment]
Comments welcome. If you do not want to allow Github API access for [giscus.app](https://giscus.app/), feel free to [directly add comments on Github](https://github.com/Sieboldianus/du.nkel.dev/discussions/13).

**Comment by jms2k** [comment]
I messed up along the way and now when trying to boot to my USB drive, I get what I'm calling the "Red screen of death". Any ideas how to get around this? I'm running a Dell PowerEdge R440. 
![Error](https://github.com/user-attachments/assets/2b3acf11-09dd-4ee4-a21d-0581e2976bd1)

**Reply by Sieboldianus** [reply]
Hey, this is not a help desk, but looking at your screen the fault is crashing on `x64sas2.efi`. This is the UEFI Option ROM (Boot Services) for the SAS card. It is either corrupted, incompatible with Dell's UEFI environment, or misconfigured.

You need to get past that screen to get into USB UEFI shell and Erase Boot Services (`sas2flash -o -c 0 -e 5`).

Try either disabling boot driver for the specific PCIe slot containing the HBA in the Dell Bios or move the card to a spare consumer motherboard, which handles these mismatched ROMs better than enterprise Dell gear.

No guarantees.
